tests/test_adaptor_strict.cc

```diff
@@ -112,11 +115,10 @@ namespace rascal {
   BOOST_FIXTURE_TEST_CASE(strict_test,
                           ManagerFixture<StructureManagerCenters>) {
     bool verbose{false};
-    int mult = 10;
-    double rc_max{mult * 0.5 + cutoff};
-    using PairManager_t = AdaptorNeighbourList<StructureManagerCenters>;
-    PairManager_t pair_manager{manager, rc_max};
-    pair_manager.update();
+    int mult = 6;
+    // double rc_max{mult * 0.5 + cutoff};
+    // auto pair_manager{make_adapted_manager<AdaptorNeighbourList>(manager,
+    // rc_max)}; pair_manager->update();
 
     for (auto i{0}; i < mult; ++i) {
       auto cutoff_tmp = i * 0.5 + cutoff;
```

Comment:
`auto cutoff_tmp {i * 0.5 + cutoff};` ?

Reply:
cutoff is defined in ManagerFixture<StructureManagerCenters>